### PR TITLE
add project js-joda

### PR DESCRIPTION
**Pull request by pithu**
add the javascript date and time lib to jsdelivr

### CODE REVIEW

**Comment by jimaek** [comment]
No tags https://github.com/pithu/js-joda/releases
the bot will fail

**Comment by pithu** [comment]
ups, sorry, i changed the package manager to npm. that the way we release new version. 

thx in advance, great stuff

File: files/js-joda/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Philipp Thuerwaechter"
+github = "https://github.com/pithu/js-joda"
+homepage = "http://pithu.github.io/js-joda/"
+description = "Immutable data and time library with domain-driven design for javascript"
+mainfile = "dist/js-joda.js"
```
Comment (jimaek) [inline files/js-joda/info.ini]: Use `js-joda.min.js` here. No dist/

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**